mm2src/coins/eth.rs

```diff
@@ -524,6 +525,23 @@ impl EthCoinImpl {
     }
 }
 
+async fn get_raw_transaction_impl(coin: EthCoin, req: RawTransactionRequest) -> RawTransactionResult {
+    let hash = H256::from_str(&req.tx_hash[2..req.tx_hash.len()])
```

Comment:
`req.tx_hash` is not guaranteed to have the `0x` prefix. It can be also empty or too short string, which will lead to panic.